notification/templates/notification/notification_list.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'layout.html' %}
+{% load static %}
+
+{% block head_end %}
+    {{ block.super }}
+    <link href="{% static 'notification/css/notification.css' %}" rel="stylesheet"/>
+{% endblock %}
+
+{% block content %}
+    {% csrf_token %}
+    <div class="row mt-4">
+        <div class="jumbotron col">
+            <h1 class="display-4">Notifications</h1>
+            <hr />
+            <p>If you would like to change the way DAISY handles your notifications, please update your <a href="{% url 'notifications_settings' %}">notifications settings</a>.</p>
+            {% if not show_dismissed %}
+                <p>By default, dismissed notifications are not shown. You can display dismissed notifications by clicking <a href="{{ request.path }}?show_dismissed=true">here</a></p>
+            {% endif %}
```

Comment:
would be convenient for the user to have an easy way to hide the dismissed notification.
Adding a similar text at the one above but with a link for hiding them for instance.

Reply:
I added that in latest commit